opacus/optimizers/optimizer.py

```diff
@@ -1,12 +1,37 @@
 from __future__ import annotations
 
-from typing import Callable, List, Optional
+from typing import Callable, List, Optional, Union
 
 import torch
 from torch import nn
 from torch.optim import Optimizer
 
 
+def _label_processed(obj: Union[torch.Tensor, List[torch.Tensor]]):
```

Comment:
nit: `_mark_as_processed` ? 
I read this as the class label was processed in the first reading.